skimage/color/colorconv.py

```diff
@@ -842,6 +868,9 @@ def gray2rgb(image, alpha=None):
             is_rgb = True
 
     if is_rgb:
+        warn('RGB and RGBA images conversion is now deprecated. In version '
```

Comment:
@hmaarrfk, and for this one too?

Reply:
Yeah. You could include those.

Just magic behavior change is hard to come to a convenus to. So the smaller the change, the easier it is to process